Examples/rc-maestro/Project.swift

```diff
@@ -31,7 +19,9 @@ let project = Project(
                         "UIColorName": "",
                         "UIImageName": "",
                     ],
-                    "REVENUECAT_API_KEY": "$(REVENUECAT_API_KEY)"
+                    "REVENUECAT_API_KEY": "$(REVENUECAT_API_KEY)",
+                    "REVENUECAT_PROXY_URL_SCHEME": "$(REVENUECAT_PROXY_URL_SCHEME)",
+                    "REVENUECAT_PROXY_URL_HOST": "$(REVENUECAT_PROXY_URL_HOST)"
```

Comment:
Question: wouldn't this be covered by the xcconfig?

Reply:
mmm good question, I'll try it out